recipe/conda_build_config.yaml

```diff
@@ -52,13 +52,15 @@ pin_run_as_build:
 arpack:
   - 3.5
 boost:
-  - 1.65.1
+  - 1.66.0
+boost-cpp:
+  - 1.66.0
 bzip2:
   - 1.0
 cairo:
   - 1.14
 curl:
-  - 7.54
+  - 7.44
```

Comment:
We seem to have an [older pinning of `curl`]( https://github.com/conda-forge/conda-forge.github.io/blob/61cdd7fd0374c07e95afdcbd414c7985b141e4b3/scripts/pin_the_slow_way.py#L70 ).